finalPrep/modZeroHero.js

```diff
@@ -2,40 +2,64 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = "Mr. Incredible"
+var specialAbility = "strength"
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = ("Hello " + heroName)
+var catchPhrase = (specialAbility + " unlike anything you've seen!")
 
 // Declare two variables - power AND energy - set to integers
+var power = 100
+var energy = 98
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
+var fullPower = (power * 500)
+var fullEnergy = (energy + 150)
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true
+var identityConcealed = false
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["bad guys", "evil guys","surley guys"]
+var sidekicks = ["violet", "jack jack", "Dax"]
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
 // Write some code to add a new archEnemy to the archEnemies array
-
+archEnemies.push("sketchy dudes");
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemies);
 // Remove the first sidekick from the sidekicks array
-
+sidekicks.shift()
 // Print the sidekicks array to console to ensure you added a new sidekick
-
+console.log(sidekicks);
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
-
+var dangerLevel = 70
+var saveTheDay = "I'm just doing my job"
+var badExcuse = "I've got a world superheroe meeting"
+
+function assessSituation() {
+  if (dangerLevel < 50 && dangerLevel > 10) {
+    console.log(saveTheDay);
+  } else if (dangerLevel < 10 ) {
+    console.log("Meh. Hard pass.");
+  } else {
+    console.log(badExcuse);
+  }
+}
+assessSituation();
 // Your function should include an if/else statement that meets the following criteria
 //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
```

Comment:
What will happen in your assessSituation functio if the `dangerLevel == 10` ? Make sure you account for ALL cases (happy and sad paths) specified in the directions.